docs/manual-EN/1.overview/2.quick-start/1.get-started.md

```diff
@@ -1,306 +1,414 @@
-| 200       |
--------------
-| 202       |
--------------
+===============
+| 101         |
+---------------
+| 102         |
+---------------
 ```
 
-Q2. Find the vertexes that 201 follows, whose ages are greater than 17.
-Return their name, age and gender, and alias the columns as Friend, Age and Gender, respectively.
+**Example 2**. Find the vertex that 100 follows, whose age is greater than 35.
+Return his name and age, and set the column names to **Teammate** and **Age** respectively.
+
+Enter the following statement:
 
 ```ngql
-nebula> GO FROM 201 OVER follow WHERE $$.student.age >= 17 YIELD $$.student.name AS Friend, \
-  $$.student.age AS Age, $$.student.gender AS Gender;
-=========================
-| Friend | Age | Gender |
-=========================
-|   Jane |  17 | female |
--------------------------
+nebula> GO FROM 100 OVER follow WHERE $$.player.age >= 35 YIELD $$.player.name AS Teammate, $$.player.age AS Age;
 ```
 
-You can use backslash (`\`) as line break;
+The following information is returned.
+
+```
+==================
+| Teammate | Age |
+==================
+| Tony     | 36  |
+------------------
+```
 
-* `YIELD` specifies what values or results you might want to return from query.
+**Note**:
+
+* `YIELD` specifies what values or results you might want to return from the query.
 * `$^` represents the source vertex.
 * `$$` indicates the target vertex.
 
-Q3. Find the chosen courses and corresponding grades of students followed by 201.
+**Example 3**. Find the team which is served by the player who is followed by `100`.
 
-```ngql
--- By pipe
-nebula> GO FROM 201 OVER follow yield follow._dst as id \
-  | GO FROM $-.id OVER choose YIELD $^.student.name AS Student,\
-  $$.course.name AS Course, choose.grade AS Grade;
-=============================
-| Student |  Course | Grade |
-=============================
-|  Monica |    Math |     6 |
------------------------------
-|  Monica | English |     3 |
------------------------------
-|    Jane | English |     3 |
------------------------------
-
--- By temporary variable
-nebula> $var=GO FROM 201 OVER follow yield follow._dst as id; \
-  GO FROM $var.id OVER choose YIELD $^.student.name AS Student,\
-  $$.course.name AS Course, choose.grade AS Grade;
-=============================
-| Student |  Course | Grade |
-=============================
-|  Monica |    Math |     6 |
------------------------------
-|  Monica | English |     3 |
------------------------------
-|    Jane | English |     3 |
------------------------------
-```
-
-* `|` denotes a pipe. The output of the formal query acts as input to the next one follow a pipeline.
-* `$-` refers to the input stream.
+There are two ways to get the same result. First, we can use a `pipe` to retrieve the team. Then we use a `temporary variable` to retrieve the same team.
+
+Enter the following statement with a `pipe`:
+
+```
+GO FROM 100 OVER follow YIELD follow._dst AS id | GO FROM $-.id OVER serve YIELD $$.team.name AS Team;
```

Comment:
backslash as well. looks too long for my screen

![image](https://user-images.githubusercontent.com/50101159/69235469-70af6c00-0bcc-11ea-9098-1ce0923d419e.png)